pkg/cpio/cpio.go

```diff
@@ -96,3 +96,9 @@ func (w Writer) WriteRecords(files []Record) error {
 func (w Writer) WriteTrailer() error {
 	return w.WriteRecord(TrailerRecord)
 }
+
+func MakeReproducible(files []Record) {
```

Comment:
I'm still not sure what this is making reproducible, but I'm okay with the change overall.

What's the issue with mtime? Something with flash? Is the issue that when CreateFile recreates the file, it doesn't set the same mtime?